src/test/java/bio/terra/common/fixtures/ConnectedOperations.java

```diff
@@ -490,20 +516,38 @@ public void retryAcquireLockIngestFileSuccess(
         TimeUnit.SECONDS.sleep(5); // give the flight time to fail a couple of times
         datasetDaoUtils = new DatasetDaoUtils();
         String[] sharedLocks = datasetDaoUtils.getSharedLocks(datasetDao, UUID.fromString(datasetId));
+        if (retryType.equals(RetryType.lock)) {
+            assertEquals("no shared locks after first call", 0, sharedLocks.length);
+        } else {
+            assertEquals("Acquire shared locks after first call", 1, sharedLocks.length);
+            logger.info("shared lock acquired: datasetId: {}", sharedLocks[0]);
+        }
 
-        // Remove insertion of shared lock fault
-        configService.setFault(faultToInsert.name(), false);
-
-        assertEquals("no shared locks after first call", 0, sharedLocks.length);
+        if (removeFault) {
+            // Remove insertion of shared lock fault
```

Comment:
wordo - not only shared lock, right?

Reply:
Right - it should be shared lock and unlock. (Exclusive lock tests don't reference this function.)